changelog/10589.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Allow users to override repr maxsize value (240) by defining
```

Comment:
This should be considered a new feature, so please rename this file to `10589.feature.rst`.